app/src/main/java/com/myvetpath/myvetpath/CreateSubActivity.java

```diff
@@ -30,6 +65,47 @@ protected void onCreate(Bundle savedInstanceState) {
         setSupportActionBar(toolbar);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
+        view_subs_activity = new Intent(this, ViewSubsActivity.class);
+
+        dbHandler = new MyDBHandler(this);
+        final Submission newSub = new Submission();
+
+        //For displaying the current date
+        final SimpleDateFormat simpleDateFormat = new SimpleDateFormat();
+
+        //initialize submission elements
+        title_et = findViewById(R.id.sub_title);
+        save_draft_button = findViewById(R.id.save_draft_btn);
+        save_draft_button.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                hideSoftKeyboard();
+                long curDate = Calendar.getInstance().getTime().getTime();
+                newSub.setCaseID(NULL);
+                newSub.setTitle(title_et.getText().toString());
+                newSub.setStatusFlag(0);
+                newSub.setDateOfCreation(curDate);
+                //Display confirmation Toast
+                String content = title_et.getText().toString() + " Saved";
+                Toast testToast = Toast.makeText(getApplicationContext(), content, Toast.LENGTH_LONG);
+                testToast.show();
+                dbHandler.addSubmission(newSub);
```

Comment:
Later on we may also want to add a dialog box here to keep it consistent with the submit button.